war/src/main/webapp/css/responsive-grid.css

```diff
@@ -1275,7 +1275,7 @@ h1.build-caption.page-headline {
  */
 @media (min-width: 1800px) {
   .container {
-    width: 1760px;
+    width: 100%
```

Comment:
What's the reason for changing the width  of these containers?

Reply:
It seems to me that the container class is used to group form items, a bit alike the `.form-group` class of bootstrap. Is this correct?